terraform/hcl/applicable/req_sw_terraform_aws_api_gateway_tls_version.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_api_gateway_domain_name" "vulnerable_example" {
```

Comment:
[comment]: <> (FrogbotReview Iac 1 1 4 2 "vulnerable_example" /terraform/hcl/applicable/req_sw_terraform_aws_api_gateway_tls_version.tf)

## ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableLowSeverity.png)<br>     low Infrastructure as Code Vulnerability 🛠️
	
Finding: security_policy!=TLS_1_2 was detected

### 👇 Details

Resource `aws_api_gateway_domain_name` should have `security_policy=TLS_1_2`

Vulnerable example -
```
resource "aws_api_gateway_domain_name" "vulnerable_example" {
  # security_policy is not set
}
```

Secure example -
```
resource "aws_api_gateway_domain_name" "secure_example" {
  security_policy = "TLS_1_2"
}
```	

[[🐸 JFrog Frogbot]](https://github.com/jfrog/frogbot#readme)